Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu
+
+
+#install golang 
+
+RUN apt-get update
+RUN apt-get install -y curl && apt-get install -q -y \
+    gnupg2 \
+    git \
+    vim \
+    openssh-client \
+    curl \
+    wget
+RUN rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION 1.13.3
+
+RUN curl -sSL https://storage.googleapis.com/golang/go$GOLANG_VERSION.linux-amd64.tar.gz \
+		| tar -v -C /usr/local -xz
+
+ENV PATH /usr/local/go/bin:$PATH
+RUN mkdir -p /go/src /go/bin && chmod -R 777 /go
+ENV GOROOT /usr/local/go
+ENV GOPATH /go
+ENV PATH /go/bin:$PATH
+
+# install oc
+RUN wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz && \
```

Comment:
Why 3.11? Is Tekton/Pipelines supported on 3.11?

Reply:
@sthaha we better remove Dockerfile as we don't support it, what do you say?

@ppitonak Good catch that was just work around as I said, basically we are using oc client  in some scenarios so technically it should work. But good catch Need to update that to latest version.

Reply:
@praveen4g0 what do you mean by "don't support" it. I suggest to promote using containers for local execution and development of tests.

Reply:
by don't support he means that it is a problem can solve later. The value currently is in writing tests and be able to run tests which you now can locally without having to rely on docker. Being able to use a container that has all tools packages IMO is only a good to have at this point.